docs/customer_management/cp_configuration.md

```diff
@@ -29,7 +29,8 @@ ibexa:
 ## Roles and Policies
 
 You can add custom roles to your installation
-by adding them to the configuration file `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml`:
+by listing them under the `ibexa.site_access.config.default.corporate_accounts.roles` key
+(it overwrites the default list set in `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml`, the following example keep them by redeclaring them):
```

Comment:
@DominikaK Test done, it doesn't add, it replaces. I'm not satisfied with my wording. Can you help me, please?

Reply:
Done in 0e63874a3c9db3e891de85e1ec877437d62bef18